daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -241,7 +241,7 @@ object Speedy {
       commitLocation = None
       val seedWithTime = for {
         time <- ptx.submissionTime
-        currentSeed <- ptx.context.contextSeed
+        currentSeed <- ptx.context.nextChildrenSeed
```

Comment:
Here we may not have any more `contextSeed`, so we take another seed.
Does not really matter as this is called only when interpretation scenario as long the resulting `newSeed` is fresh.